cmd/operator-sdk/new/cmd.go

```diff
@@ -357,6 +359,12 @@ func doHelmScaffold() error {
 		Repo:               apiFlags.HelmChartRepo,
 	}
 
+	chartsDir := filepath.Join(cfg.AbsProjectPath, helm.HelmChartsDir)
+	err := os.MkdirAll(chartsDir, 0755)
+	if err != nil {
+		return fmt.Errorf("failed to create helm-charts directory: %v", err)
+	}
+
```

Comment:
It was added here for we do not need to duplicated the code in `CreateChart` and use it for the plugin as well.